amazon-s3/5.7/modules/ROOT/pages/index.adoc

```diff
@@ -10,35 +10,64 @@ Instant access to the Amazon S3 API enables seamless integrations between Amazon
 AWS SDK for Java provides a Java API for AWS infrastructure services.
 The Amazon S3 connector is built using the SDK for Java.
 
-Release Notes: xref:release-notes::connector/amazon-s3-connector-release-notes-mule-4.adoc[Amazon S3 Connector Release Notes] +
-Exchange: https://anypoint.mulesoft.com/exchange/com.mulesoft.connectors/mule-amazon-s3-connector[Amazon S3 connector in Anypoint Exchange]
+For compatibility information and fixed issues, see the xref:release-notes::connector/amazon-s3-connector-release-notes-mule-4.adoc[Amazon S3 Connector Release Notes].
+
+For a major or minor release, see xref:amazon-s3-connector-upgrade-migrate.adoc[Upgrade and Migrate].
```

Comment:
Yes! I know some other connectors (Amazon SQS, Salesforce) have a section in the Upgrade topic for what has changed in each minor release. Since it isn't there for S3, I'll change it to what you said. I added a story for the Upgrade topic for this and there is a PR with changes for the 5.7 release (right now it is showing upgrade changes for 5.5). I can verify the Upgrade PR with the devs.